kong/cli/utils/services.lua

```diff
@@ -13,8 +13,8 @@ _M.STATUSES = {
 -- Services ordered by priority
 local services = {
   require "kong.cli.services.dnsmasq",
-  require "kong.cli.services.nginx",
-  require "kong.cli.services.serf"
+  require "kong.cli.services.serf",
+  require "kong.cli.services.nginx"
```

Comment:
I think you will like this @mars - although terminating the foreground process will not terminate all the other services (like < v0.6.x was erroneously doing). As opposed to #1007 which terminates the other process when the main one is terminated (but not really an issue if terminating means shutting down the whole Docker container). 

Basically this would allow the older behavior to work again. Still, not a replacement of supervisor (or similar solutions).